translations/ui-bulk-edit/en.json

```diff
@@ -9,14 +9,15 @@
   "app-view": "Bulk Edit: In app - View",
   "app-edit": "Bulk Edit: In app - Edit",
   "app-delete": "Bulk Edit: In app - Delete",
-
+  "logs.view": "Bulk edit - Can view logs",
```

Comment:
@UladzislauKutarkin, for permission translations to be picked up, two things are necessary:
1. In `package.json`, the `permissionName` field must start with a prefix of the application name followed by a dot, e.g. `ui-foo.whatever.view`. (`package.json` will be used to generate a module descriptor which will be POSTed to okapi; including the application name as a prefix is a convention that allows us to figure out which permission were created by which modules.)
1. In the translation file, the corresponding key must start with the prefix `permission.` instead of the application-name, e.g. `permission.whatever.view`. (Translation files are post-processed to have the application name prefixed to each key, saving us from having to do that manually.)